packages/tooling/fast-tooling-react/src/form/controls/control.section.tsx

```diff
@@ -28,6 +28,7 @@ import {
     SchemaSetValidationMessageRequest,
     TreeNavigationItem,
 } from "@microsoft/fast-tooling";
+import { FormStrings } from "@microsoft/fast-tooling-react/src/form/form.props";
```

Comment:
Same issue with the import pathing here, should be a relative path.

Reply:
Done